extensions/wikia/GlobalNavigation/styles/GlobalNavigation.scss

```diff
@@ -91,5 +92,142 @@
 	display: none;
 }
 
+.global-navigation-2016 {
+	.global-nav-search-input-wrapper .autocomplete-container,
+	.notifications-container .notifications-container,
+	.account-navigation-container .user-menu,
+	.AccountNavigation .UserLoginDropdown {
+		margin-top: 1px;
+	}
+
+	.wikia-logo {
+		align-items: flex-end;
+		display: flex;
+		text-decoration: none !important;
+	}
+
+	.wikia-logo__subtitle {
```

Comment:
I supposed so. However, we're not using BEM and our guidelines encourage dashed-case for (s)css.